pkg/statesolver/buildkit.go

```diff
@@ -122,8 +122,7 @@ func (s BuildkitSolver) FileExists(
 	filepath string,
 	source *builddef.Context,
 ) (bool, error) {
-	// @TODO: switch to StatFile once it got fixed upstream (it currently fails the hard way if the given filepath does not exist).
```

Comment:
does it now require a specific version of a go dependency ?

Reply:
Nope actually there's no bug, unlike what I thought at first ^^